docs/sphinx/manuals/roc_recv.rst

```diff
@@ -138,31 +148,37 @@ Output to the default ALSA device:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -d alsa
+    $ roc-recv -vv -o alsa://default -s rtp+rs8m::10001 -r rs8m::10002
 
 Output to a specific PulseAudio device:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -d pulseaudio -o <device>
+    $ roc-recv -vv -o pulse://alsa_input.pci-0000_00_1f.3.analog-stereo -s rtp+rs8m::10001 -r rs8m::10002
 
 Output to a file in WAV format:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -o ./file.wav
+    $ roc-recv -vv -o file:./output.wav -s rtp+rs8m::10001 -r rs8m::10002
 
 Output to stdout in WAV format:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -d wav -o - > ./file.wav
+    $ roc-recv -vv -o file:- -f wav -s rtp+rs8m::10001 -r rs8m::10002 > ./output.wav
```

Comment:
Could you also add an example for a non-special file without an extension:

```bash
roc-recv -vv -o file:./file -f wav -s rtp+rs8m::10001 -r rs8m::10002
```

Reply:
Done.